### DISCUSSION TITLE
Add year view

### DISCUSSION THREAD

**Discussion by h4r1337** [davish/obsidian-full-calendar#425] [Feature Requests]
Hey Davish,
Currently your plugin have month, week and day view. Can you also add a Year view also.
This will be helpful for those who are trying to make recurring events for months.

**Comment by Bulwagga** [comment]
This would also be **super helpful** for our gardening calendar.  Thank you!

**Comment by arnaudmm** [comment]
Having a yearly view or quaterly view would be great to track some event occurences 😇
I can see it's possible with the lib you use : https://fullcalendar.io/docs/multimonth-grid
If someone is good enough to open a PR, it would be great 🙏
